charts/monasca/values.yaml

```diff
@@ -169,6 +207,13 @@ kafka:
   service:
     type: ClusterIP
     port: 9092
+  resources:
+    requests:
+      memory: 2Gi
```

Comment:
The limits also include cache, so I'm assuming 1GiB heap + FS cache + other JVM memory